pkg/scaling/executor/scale_scaledobjects.go

```diff
@@ -272,6 +275,13 @@ func (e *scaleExecutor) scaleToZeroOrIdle(ctx context.Context, logger logr.Logge
 
 			e.recorder.Eventf(scaledObject, corev1.EventTypeNormal, eventreason.KEDAScaleTargetDeactivated,
```

Comment:
This one and the new one are duplicated. We should merge these two into one event. eventEmitter.Emit() includes emitting k8s event.